sourcemash/api/feeds.py

```diff
@@ -10,10 +10,10 @@
 from sourcemash.models import Feed
 
 feed_fields = {
+    'id': fields.Integer,
```

Comment:
We never use the URI anymore since we define the url paths on the backbone models. Id is necessary now instead.